doc/whatsnew/fragments/8714.false_negative

```diff
@@ -1,3 +1,5 @@
-Emit ``assignment-from-none`` for calls to builtin methods like ``dict.update()``.
+Emit ``assignment-from-no-return`` for calls to builtin methods like ``dict.update()``.
+(Calls to ``list.sort()`` now raise ``assignment-from-no-return``
+rather than ``assignment-from-none`` for consistency.)
```

Comment:
You added that 2 days ago in https://github.com/pylint-dev/pylint/pull/8809, right ? We probably don't need to advertise the change as it's unreleased.

Reply:
```suggestion
```

Reply:
Actually, it's changed in this PR. In #8889 I added all the other methods besides list.sort() to the existing `assignment-from-none`, which is only correct if we were going to merge the messages. :-)

Reply:
list.sort() **only** was special-cased in #5738